### DISCUSSION TITLE
Tab is very undescriptive

### DISCUSSION THREAD

**Discussion by NinjaPerson24119** [open-webui/open-webui#1228] [General]
Maybe not quite a bug, but I continuously struggle to find Open WebUI's tab because neither the icon or title are obvious.

![image](https://github.com/open-webui/open-webui/assets/32471098/892bc83b-6d2f-43f0-8883-3e536129fe36)

It looks like some random blog. I'd much prefer if there
- were a but more color, or graphic. I'm just now realizing it's supposed to say "OI". At a glance it looked like the I was supposed to be something else, but cropped off the right. Might be just my own eyes.
- I would much prefer if the tab just said "Open Web UI" or if that was configurable. I avoid opening multiple tabs. The web app has a vertical tab menu anyway.

**Comment by justinh-rahb** [comment]
We can revert #992 but it was requested [here](https://github.com/open-webui/open-webui/issues/957#issuecomment-1973395956). If we don't want to _unfix_ that, we would indeed need an option.

**Comment by NinjaPerson24119** [comment]
Ah ok. I can definitely see that being useful for a multi-tab flow.

I just tried out ChatGPT as a reference and it also changes the tab title by the chat.

Maybe the icon just doesn't seem that obvious to me with the neutral colors and plain text.
